CMakeLists.txt

```diff
@@ -802,7 +814,7 @@ install(FILES omnisci.thrift DESTINATION ".")
 install(FILES common.thrift DESTINATION ".")
 install(FILES QueryEngine/serialized_result_set.thrift DESTINATION "QueryEngine/")
 
-if(NOT PREFER_STATIC_LIBS)
+if(NOT PREFER_STATIC_LIBS AND NOT ENABLE_CONDA)
```

Comment:
Thumbs up!
Btw, is `install(FILES ${Boost_LIBRARIES} DESTINATION ThirdParty/lib)` really needed?